aerogear/apollo-voyager-ionic-example
# JS-SDK integration

Replace Apollo Client by AeroGear Sync

## Verification steps:
Please follow the README for running the project. 
Check basing functionalities.
Check window.localstorage for offline queue.

More advanced tests will be performed in separate task.
